src/umpire/strategy/DynamicPool.cpp

```diff
@@ -115,7 +120,7 @@ DynamicPool::findFreeBlock(std::size_t bytes) const
   return iter;
 }
 
-void* DynamicPool::allocateFromResource(std::size_t bytes)
+void* DynamicPool::allocateFromStrategy(std::size_t bytes)
```

Comment:
@jdahm and @davidbeckingsale, I thought I  asked this question before, but I cannot find it now.  Why are we changing the name from `allocateFromResource` to `allocateFromStrategy`?  I thought that `allocateFromResource` was more clear as it at least indicated that it was performing the allocation at least from something that is close to the actual `Resource` than the calling object.  Since the calling object is already a Strategy, it is confusing to me seems recursive.

If `allocateFromResource` isn't right, can we come up with a different, less confusing name?